### PR TITLE
Deadly skills

### PR DESCRIPTION
**Pull request by Szczaw**
From deadly Vipers to master it contains changes in introduction.md

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 This is my introduction to dojo_rules.
```
Comment (codeschool-kiddo) [inline introduction.md:1]: Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

**Comment by Szczaw** [comment]
Merging master with my branch deadly cos tam

**Comment by Szczaw** [comment]
"Killing history using git rebase"

File: introduction.md (hunk 1 above)
Comment (Szczaw) [reply introduction.md:1]: I am glad you are writting

**Pull request merged**